le_lambda.py

```diff
@@ -27,6 +29,7 @@
 ENDPOINT = '{}.data.logs.insight.rapid7.com'.format(REGION)
 PORT = 20000
 TOKEN = os.environ.get('token')
+SALT = os.environ.get('salt')
```

Comment:
@cmfcruz are you configuring this via the [CFN template parameter](https://github.com/product-os/environment-production/blob/master/templates/legacy/s3-notification-configuration.yml#L146-L148)?

Reply:
@ab77 , yes. My intention is to use the same salt across all of the logs. The salt string is defined on the top-level CloudFormation template and passed to the nested CFN stacks as a parameter.